src/writers/file_log_writer/infix_filter.rs

```diff
@@ -27,7 +27,7 @@ impl InfixFilter {
             #[cfg(test)]
             InfixFilter::StartsWth(s) => infix.starts_with(s),
             InfixFilter::Equls(s) => infix.eq(s),
-            InfixFilter::None => false,
+            InfixFilter::None => true,
```

Comment:
@emabee I was not sure about semantics of None variant, should it filter all or none? If former then I would create a new variant.